consts.js

```diff
@@ -106,7 +106,12 @@ var thirteenStrings = [
     "wkluwhhq",
     "Wkluwhhq",
     "WKLUWHHQ",
-
+   
+    //md5 hash
+    "c51ce410c124a10e0db5e4b97fc2af39"
+    //also md5 hash but UPPERCASE
```

Comment:
you remembered to account for uppercase?? :O 

You'll go far as a programmer :)

Reply:
I know that since the very first word I said is "thirteen" but not "mama".